frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_system_view.dart

```diff
@@ -18,15 +19,16 @@ class _SettingsFileSystemViewState extends State<SettingsFileSystemView> {
   @override
   Widget build(BuildContext context) {
     return ListView.separated(
+      shrinkWrap: true,
       itemBuilder: (context, index) {
         if (index == 0) {
           return SettingsFileLocationCustomzier(
             cubit: _locationCubit,
           );
         } else if (index == 1) {
-          // return _buildExportDatabaseButton();
+          return const SettingsExportFileWidget();
```

Comment:
The first if statement returns if its condition is true so the `else if` isn't necessary.

```
        if (index == 0) {
          return SettingsFileLocationCustomzier(
            cubit: _locationCubit,
          );
        } 
        if (index == 1) {
          return const SettingsExportFileWidget();
        }
```

Reply:
To avoid all of the brackets from the if statements, it might be worth it to use a `switch`.

```dart
switch (index) {
  case 0:
    return SettingsFileLocationCustomzier(
      cubit: _locationCubit,
    );
  case 1: 
    return const SettingsExportFileWidget();
  default:
    return const SizedBox.shrink();
}
```

Reply:
If the upgrade to 3.10 lands we can use patterns instead 
https://dart.dev/language/patterns